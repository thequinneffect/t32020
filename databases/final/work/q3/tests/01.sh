psql music -c 'select * from q3 order by ninstruments desc, performer;'
