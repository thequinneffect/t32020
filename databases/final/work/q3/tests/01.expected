    performer    | ninstruments 
-----------------+--------------
 Danny Jones     |            9
 Sam Strummer    |            9
 Bernie Jaeger   |            7
 Bobby Storm     |            7
 Dave Jones      |            7
 Eric Page       |            7
 Mick Smith      |            7
 Ozzy Wheeler    |            7
 Robert Smith    |            7
 Robert Storm    |            7
 Weasel Osbourne |            7
(11 rows)
